dashboard-app/src/components/StackedBarChart/Chart.tsx

```diff
@@ -65,14 +67,15 @@ const Chart: FunctionComponent<Props> = (props) => {
     title,
     isAllLegendDataInactive,
     noActiveLegendText,
+    unit,
```

Comment:
I don't know whether #183 was in part motivated by this kind of change, but my specific suggestion here would be that, it is actually quite reasonable for a "Chart" to be told what units the quantities it is displaying are measured in. 

Depending on your perspective about how generic this component should be, it is perhaps less reasonable for it to be given specific knowledge about hours/days (as is happening in `getStackedBarChartOptions`).

My suggestion would be that rather than `unit` being passed down, instead we pass down `xAxisUnit: string` and `yAxisUnit: string`. This is more natural for a chart component, more generic, and allows us to label axes with months if we wish to do so later on.

So we would pass down `{ yAxisUnit: 'hrs' }` or `{ yAxisUnit: 'days' }` from the parent. I haven't yet figured whether either prop should be optional.

Reply:
`getStackedBarChartOptions` is already very tied to the particular type of graph we want to create. If we do end up displaying multiple types of the graph then we should split out the options so that we are mutating `Chart` with our default config, then generate specific chart options via a funcs similar to this.